examples/plugin.rs

```diff
@@ -0,0 +1,28 @@
+use wasmer_runtime::{func, imports, instantiate};
+use wasmer_runtime_core::vm::Ctx;
+use wasmer_wasi::generate_import_object;
+
+static PLUGIN_WASM: &'static [u8] = include_bytes!("plugin-for-example.wasm");
```

Comment:
how about instead of statically importing the bytes, we load them at runtime? We already do this in our rust example project. I think loading the wasm at runtime more accurately represents the behavior of a "plugin" as plugins are dynamically loaded (to me).